linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/Knife4jConfig.scala

```diff
@@ -0,0 +1,70 @@
+package org.apache.linkis.server.conf
+
+import org.springframework.context.annotation.Bean
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+import com.github.xiaoymin.knife4j.spring.annotations.EnableKnife4j
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+import springfox.documentation.builders.ApiInfoBuilder
+import springfox.documentation.builders.PathSelectors
+import springfox.documentation.builders.RequestHandlerSelectors
+import springfox.documentation.service.ApiInfo
+import springfox.documentation.spi.DocumentationType
+import springfox.documentation.spring.web.plugins.Docket
+import springfox.documentation.swagger2.annotations.EnableSwagger2WebMvc
+
+/**
+- 启动knife的apidoc的步骤很简单
+- 1,打开application-linkis.yml,配置knife4j.enable=true
+- 2,打开linkis.properties，配置ds.linkis.test.mode=true
+- 3，启动服务之后，通过微服务地址http://ip:port/api/rest_j/v1/doc.html访问
+ */
+@EnableSwagger2WebMvc
```

Comment:
Is wds.linkis.test.mode not used?

Reply:
there are two ways to enable knife4j:
 * it is very easy to enable knife when you want to check the apiDoc
 * you can follow these steps to enable
 * 1, open application-linkis.yml and set knife4j.production=false
 * 2, open linkis.properties and set ds.linkis.test.mode=true
 * 3, restart the service and you can visit http://ip:port/api/rest_j/v1/doc.html
 *
 * or you can use apidoc by following steps  without enable ds.linkis.test.mode
 * 1, open application-linkis.yml and set knife4j.production=false
 * 2,vi linkis.propertes ,and set wds.linkis.server.user.restful.uri.pass.auth=/api/rest_j/v1/doc.html,/api/rest_j/v1/swagger-resources,/api/rest_j/v1/webjars,/api/rest_j/v1/v2/api-docs
 * 3, in your browser,add dataworkcloud_inner_request=true, bdp-user-ticket-id's value and  workspaceId's value
 * 4, restart the service and you can visit http://ip:port/api/rest_j/v1/doc.html